### ISSUE TITLE
ReferenceError Game_BattleMap is not defined

### ISSUE THREAD

**Issue by ghost** [je-can-code/rmmz-plugins#3]
I seem to get this error and I'm not really sure why.
I have tried reordering the plugins but it doesn't seem to fix the issue.
Here's a screenshot of my plugins in case it helps, I might have done something wrong but I'm not really sure why I get this error,

![error](https://user-images.githubusercontent.com/78994214/111770185-32c87200-88fe-11eb-9920-4d48e0df65f6.png)

**Comment by je-can-code** [comment]
My apologies @lurkingaround !
It was developed in tandem with intention to use alongside JABS as well. I didn't do a lot of testing for non-ABS users, thus you get that error.

I've updated `master` to include a fix that should prevent that from happening again, along with a few other features.

I tested it locally, but why don't you go ahead and download it yourself and try again? (or just copy pasta the plugin over your own locally since you seem savvy with github at least) If its still busted, we can talk more. If it isn't, I'll close the issue 💯
